bin/functions

```diff
@@ -76,19 +77,21 @@ function retrieve-sapmachine() {
     | select(.name | startswith("sapmachine-jdk"))
     | select(.name | endswith("linux-x64_bin.tar.gz") or endswith("osx-x64_bin.tar.gz"))
     | {
-      binary_link: .browser_download_url,
-      checksum_link: (.browser_download_url | sub("tar\\.gz$"; "sha256.txt")),
+      package: {
+        binary_link: .browser_download_url,
+        checksum_link: (.browser_download_url | sub("tar\\.gz$"; "sha256.txt"))
+      },
       os: (if .name | endswith("osx-x64_bin.tar.gz") then "mac" else "linux" end),
       architecture: "x64",
-      openjdk_impl: "hotspot",
+      jvm_impl: "hotspot",
       heap_size: "normal"
     }
   ]}
 ]'
     # shellcheck disable=SC2046
     if [[ ! -r "${cache_file}" ]] || [[  $(set -- $(${STAT}) && echo "${1}") -le $(( $(date +%s) - 3600)) ]]
     then
-        curl "${args[@]}" 'https://api.github.com/repos/SAP/SapMachine/releases' -o "${cache_file}"
+        curl "${args[@]}" -L 'https://api.github.com/repos/SAP/SapMachine/releases' -o "${cache_file}"
```

Comment:
Any reason -L should not be included in `args`?

Reply:
@halcyon No specific reason, to be honest. This specific change was caused by reverting #55 in this PR.